tools/system_libs.py

```diff
@@ -422,7 +423,8 @@ def build_objects(self, build_dir):
     """
     commands = []
     objects = []
-    cflags = self.get_cflags()
+    source_dir = utils.path_from_root()
+    cflags = self.get_cflags() + [f'-fdebug-prefix-map={source_dir}=/emsdk/emscripten']
```

Comment:
Actually I wonder if this makes sense to do on the buildbot only somehow?  If we are running locally it seems like we might as well use the actual path?